packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -97,6 +97,13 @@ const CreateAdForm = () => {
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
                     const is_sell_advert = values.type === buy_sell.SELL;
 
+                    const invalidAmountError = Object.values(errors).includes('Enter a valid amount');
```

Comment:
Why do we need this @artsemkavaleu-deriv? The same error is available in the store.

Reply:
We need to make a prioritisation of error appearing, 'Enter a valid amount' error should appear first and without any other errors, so that's why i do that

Reply:
As discussed, please remove this now @artsemkavaleu-deriv.